security/advancedtls/advancedtls_test.go

```diff
@@ -97,47 +97,30 @@ func TestClientServerHandshake(t *testing.T) {
 		// even setting vType to SkipVerification. Clients should at least provide
 		// their own verification logic.
 		{
-			"Client_no_trust_cert_Server_peer_cert",
-			nil,
-			nil,
-			nil,
-			nil,
-			nil,
-			SkipVerification,
-			true,
-			false,
-			false,
-			[]tls.Certificate{serverPeerCert},
-			nil,
-			nil,
-			nil,
-			nil,
-			CertAndHostVerification,
-			true,
+			desc:                       "Client_no_trust_cert_Server_peer_cert",
+			clientVType:                SkipVerification,
+			clientExpectCreateError:    true,
+			clientExpectHandshakeError: false,
+			serverMutualTLS:            false,
```

Comment:
You could skip fields which are set to their default values, like `clientExpectHandshakeError` and `serverMutualTLS` here, unless you think they make the test case more readable.

Reply:
+1. The whole idea behind using field names is to be able to skip fields with default values and make the code more compact :)

So basically everything with values like `false`, `nil`, `""`, `0`, etc, can be skipped.

Reply:
Done.